scripts/private/api/virus-total/Get-VirusTotalResults.ps1

```diff
@@ -0,0 +1,70 @@
+function Get-JsonResults() {
+    param(
+        $inputObject
+    )
+
+    try {
+        # we first try normal serialization
+        return $inputObject | ConvertFrom-Json
+    }
+    catch {}
+
+    try {
+        # next we try to serialize as hash table
+        return $inputObject | ConvertFrom-Json -AsHashtable
```

Comment:
> -AsHashtable may not be supported in PS5. 

You are right, it is not supported in PS5. We are however explicitly telling GitHub action to use Pwsh, so that is fine. Backwards compatibility is not a concern as these files aren't supposed to be ran on a users computer.

Also to mention, from what I remember when this was added the entire thing would fall over if `-AsHashtable` wasn't used.